runtime/vm/stringhelpers.cpp

```diff
@@ -362,8 +361,8 @@ copyStringToJ9UTF8WithMemAlloc(J9VMThread *vmThread, j9object_t string, UDATA st
 	Assert_VM_notNull(string);
 
 	U_8* result = NULL;
-	UDATA stringLength = J9VMJAVALANGSTRING_LENGTH(vmThread, string) * 3;
-	UDATA length = sizeof(((J9UTF8*)0)->length) + prependStrLength + stringLength;
+	UDATA stringLength = J9VMJAVALANGSTRING_LENGTH(vmThread, string);
+	UDATA length = sizeof(((J9UTF8*)0)->length) + prependStrLength + stringLength * 3;
```

Comment:
For clarity, please bracket `stringLength * 3`

Reply:
Fixed in b67c08c